### DISCUSSION TITLE
On change multi select livewire event

### DISCUSSION THREAD

**Discussion by umarrazapreesoft** [livewire/livewire#4828] [1. Help]
Hi everyone, good day!

I have a select of type multiple. 

```
<div class="col-md-4 mb-3">
    <label for="code" class="col-form-label-sm pt-0 required">{{ __('Owner') }}</label>
    <select class="form-control form-control-sm" name="owner_id">
        @foreach ($project->owners() as $owner)
            <option value="{{ $owner->id }}">{{ $owner->name }}</option>
        @endforeach
    </select>
    @include('includes.partials.error', ['field' => 'owner_id'])
</div> 
```

What I want is when the user changes or selects the options in the multi-select, I want the Ids of all selected options in the related livewire component. How can I achieve this?

**Comment by kaulvimal** [comment]
You can do something like [this](https://laravelplayground.com/#/snippets/e39a5f53-c77b-41e9-8dd0-5034a78e0ddf)

you can add `wire:model` in your select element and you will get all selected values in this variable.
